nall/serializer.hpp

```diff
@@ -76,10 +76,11 @@ struct serializer {
   }
 
   template<typename T> auto operator()(T& value) -> serializer& {
-    static_assert(has_serialize_v<T> || is_integral_v<T> || is_floating_point_v<T>);
+    constexpr bool integral = is_integral_v<T> || is_same_v<T, u128>;
```

Comment:
Yet another way __int128 is sketchy. I'd like to get rid of the remaining uses of 128-bit types.

https://quuxplusone.github.io/blog/2019/02/28/is-int128-integral/